### PR TITLE
jjb: correctly set the storage method

### PR DESCRIPTION
**Pull request by MaximilianMeister**
setting want_ceph is useless, as it is overwritten by storage_method

### CODE REVIEW

File: jenkins/ci.suse.de/templates/cloud-mkcloud-job-upgrade-nondisruptive-ha-template.yaml [hunk 1]
```diff
@@ -21,7 +21,7 @@
             upgrade_cloudsource=develcloud{version}
             nodenumber=5
             hacloud=1
-            want_ceph=0
+            storage_mode=none
```
Comment (bmwiedemann) [inline jenkins/ci.suse.de/templates/cloud-mkcloud-job-upgrade-nondisruptive-ha-template.yaml]: should be storage_method=none

**Comment by jsuchome** [comment]
Does `storage_method=none` imply swift? Or what will ensure there will be no ceph?

**Comment by MaximilianMeister** [comment]
> Does storage_method=none imply swift? Or what will ensure there will be no ceph?

```bash
case "$storage_method" in
  none)
    want_ceph=0
    want_swift=0
    ;;
  swift)
    want_ceph=0
    want_swift=1
    ;;
  ceph)
    want_ceph=1
    want_swift=0
    if [[ $nodenumber < 5 ]] ; then
      export nodenumber=5
    fi
    cephvolumenumber=2
    ;;
  *)
    unset want_ceph
    unset want_swift
esac
```

**Comment by jdsn** [comment]
@jsuchome the values are (see also comment of the variable _storage_method_ of the openstack-mkcloud job):
* none: mkcloud will not deploy ceph or swift at all (you have to make sure that a storage is added otherwise to the deployed cloud)
* default: mkcloud will pick the storage method (depending on other settings, like nodenumber)
* ceph: enforces deployment of ceph
* swift: enforces deployment of swift

**Comment by jsuchome** [comment]
I thinnk we want swift backend for upgrade, so we have it as glance backend

**Comment by MaximilianMeister** [comment]
@jdsn @jsuchome updated to swift

**Pull request merged**